src/endpoints/PricingTables/types.ts

```diff
@@ -36,3 +37,18 @@ export interface AddOn {
     display_name: string;
     prices: Price[];
 }
+
+export interface Row {
+    display_name: string;
+    description: string | null;
+    link: string | null;
+    area: string | null;
+    status: 'beta' | null;
+    is_key_item: boolean;
+    cells: Record<string, Cell>;
```

Comment:
I feel some uncertainty about using records here, do we need to keep special order of cells? Because objects does not guarantee the same order as it is a set.

Reply:
We don't. You will iterate through `options` and specify columns data, put columns in place. Then for every row you will just read the column value through `cells` property.

At least, that is how I understand it will be used when rendering table. Let me know if I'm missing something or I'm wrong.

Reply:
ok, let's see then